kerboscript_tests/functest31.ks

```diff
@@ -0,0 +1,38 @@
+//
+// I am not too sure if this is a feature we want to advertise yet,
+// but it does seem to be doable:
+//
+print "Testing varying args logic".
+
+// A function that accepts varying args.
+// the first arg is a number: how many more args to expect.
+// the rest of the args are just printed for testing.
+function print_var_args {
+  parameter how_many.
+  
+  // read parameters, filling a local string.
+  local my_string is "".
+  local i is 1.
+  until i > how_many {
+    parameter next_arg.
```

Comment:
This is the key part - note the use of a `parameter` statement in a conditional way - where it executes a number of times that varies.